packages/venia-ui/lib/components/OrderHistoryPage/orderHistoryPage.gql.js

```diff
@@ -0,0 +1,170 @@
+import { gql } from '@apollo/client';
+
+/*
+    This feature is being built ahead of GraphQL coverage that is landing in 2.4.1 of Magento. We're going to mock
+    the data based on the approved schema to make removing the mocking layer as seamless as possible.
+
+    @see https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/customer-orders.md
+ */
+
+/* eslint-disable graphql/template-strings */
+export const GET_CUSTOMER_ORDERS = gql`
+    query GetCustomerOrders {
+        customer {
```

Comment:
```suggestion
        customer @client {
```

Could move the directive from `orders` to here and prevent an unnecessary network request.

Reply:
Pushed without the tag on accident, fixed in 4c8f95a.

Reply:
Actually I was referencing the top-level `customer` field.